src/io/github/pedrokase/Dice.java

```diff
@@ -0,0 +1,50 @@
+package io.github.pedrokase;
+
+
+import java.util.Random;
+
+public class Dice {
+
+    private int sides, number;
+    private static Random random = new Random();
+
+    //Default dices have 6 sides and are rolled once
+    public Dice() {
+        sides  = 6;
+        number = 1;
+    }
+
+    public Dice(int sides){
+        this(sides, 1);
+    }
+
+    public Dice(int sides, int number){
+        this.sides  = (sides  == 0 ? 1 : sides);
+        this.number = (number == 0 ? 6 : number);
+    }
+
+
+    public int roll(){
+        //TODO Add +modifiers
+        int sides  = getSides(),
```

Comment:
1) This variable is being used in just one place, so no need to declare it over simply calling getSides() but...
2) Why call getter in a class that has access to its own fields? xD